index.bs

```diff
@@ -734,6 +736,13 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
         [=document.domain=]'s setter provides.
     </div>
 
+: <dfn>Non-Resident Credential</dfn>
+: <dfn>Server-side-resident Public Key Credential Source</dfn>
+:: A [=Server-side-resident Public Key Credential Source=], or [=Non-Resident Credential=] for short, is a [=public key credential
+    source=] whose [=credential private key=] is NOT stored in the [=authenticator=], [=client=] or [=client device=], but is instead 
+    encrypted (i.e. wrapped) and returned to the server as the [=credential ID=]. An authenticator is said to be using 
+    [=server-side credential storage modality=] when generating a [=Non-Resident Credential=].
```

Comment:
what I mean here is I'd change:
> An authenticator is said to be using [=server-side credential storage modality=] when generating a [=Non-Resident Credential=].

..to be..
> See also: [=server-side credential storage modality=]

..because that sentence reads like a definition, and it's already defined elsewhere.  this is a nit I suppose. one could also move the definition to be that sentence and change the current dfn sentence to reference the sentence here.

Reply:
Accepted and updated.